pagespeed/apache/apache_rewrite_driver_factory.cc

```diff
@@ -227,4 +236,18 @@ ApacheConfig* ApacheRewriteDriverFactory::NewRewriteOptionsForQuery() {
   return new ApacheConfig("query", thread_system());
 }
 
+GoogleString ApacheRewriteDriverFactory::MasterPidFilename() {
```

Comment:
It's unfortunate that we used "root" when apache and nginx use "master".  But I'll switch to "root" since the existing code uses that.